easybuild/easyconfigs/b/bigdft-suite/bigdft_suite_index_bug.patch

```diff
@@ -0,0 +1,19 @@
+diff -ruN bigdft-suite-1.9.1_orig/futile/flib/f_input_file.f90 bigdft-suite-1.9.1/futile/flib/f_input_file.f90
```

Comment:
path is missing an author and description

Reply:
adopted, thank you